batctl/Makefile

```diff
@@ -154,7 +154,7 @@ config-tables := \
 	claimtable \
 	dat_cache \
 	gateways \
-	loglevel \
```

Comment:
Is this removed intentionally, because I don't read anything about it in the commit message?

Reply:
Yes, this is the "not correctly included" part the commit message describes. The string in this list was "loglevel" but should have been "mcast_flags". And after this change, it is "mcast_flags".

Reply:
Okay, "not correctly included" sounded to me like the correct string is put in the wrong place ...
Actually, it's the wrong string put in the right place after all. :-)

Maybe just write "But the wrong string was put in the list when ..."?